docs/guide/overview.asciidoc

```diff
@@ -38,6 +36,42 @@ This data is buffered for a short period and sent on to APM Server.
 It listens on port 8200 by default and receives data from agents through a JSON HTTP API.
 Then it creates documents from that data and stores them in Elasticsearch.
 
-To visualize the data after it's sent to Elasticsearch,
-you can use the the dedicated APM UI bundled in X-Pack,
-or the pre-built open source Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
\ No newline at end of file
+*Elasticsearch* is a highly scalable open-source full-text search and analytics engine.
+It allows you to store, search, and analyze big volumes of data quickly and in near real time.
+
+*Kibana* is an open source analytics and visualization platform designed to work with Elasticsearch.
+You use Kibana to search, view, and interact with data stored in Elasticsearch indices.
+You an use Kibana to visualize APM data by utilizing the dedicated APM UI bundled in Basic license,
+or the pre-built, open source,
+Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
+
+[[concepts]]
+[float]
+=== APM Concepts
+
+*Agents*
+APM Agents capture different types of information from within their instrumented applications, known as events. Events can be <<transactions,`Transactions`>>, <<transaction-spans,`Spans`>>, and <<errors,`Errors`>>.
```

Comment:
When talking about multiple APM agents, the "agents" part is _always_ lowercase as it's not a specific product. When talking about a specific agent, e.g. the Elastic APM Node.js Agent, the word "Agent" is capitalized. But it depends on the sentence, e.g. in "use the Node.js agent for Elastic APM" the word "agent" isn't capitalized.

Reply:
Interesting - I'd like to chat more with you about this at some point.